src/components/Organization.jsx

```diff
@@ -20,17 +29,27 @@ export function Organization() {
           {organizers.map((organization) => (
             <div key={organization.name}>
               <a href={organization.site}>
-                <Image
-                  className="hover:opacity-80"
-                  src={`/images/organization/${
-                    currentTheme === 'dark'
-                      ? organization.logo
-                      : organization.logo_light ?? organization.logo
-                  }`}
-                  alt={organization.name}
-                  height={80}
-                  width={125}
-                />
+                {currentTheme == 'dark' ? (
+                  <>
+                    <Image
+                      src={`/images/organization/${organization.logo}`}
+                      alt={organization.name}
+                      height={120}
+                      width={150}
+                    />
+                    <p className="hidden">{organization.logo}</p>
+                  </>
+                ) : (
+                  <>
+                    <Image
+                      src={`/images/organization/${organization.logo_light}`}
+                      alt={organization.name}
+                      height={120}
+                      width={150}
+                    />
+                    <p className="hidden">{organization.logo_light}</p>
```

Comment:
Please don't ask me to change this line, because somehow without it it isn't working